scripts/lint/markdown-heuristic-lint.py

```
@@ -0,0 +1,320 @@
+
+Usage:
+    python3 scripts/lint/markdown-heuristic-lint.py <file> [<file> ...]
+
+Exit codes:
+    0 — no violations found
+    1 — one or more violations found
+"""
+
+import re
+import sys
+from typing import List, Optional
```

Ruff (0.15.10)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)